.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+
+name: Bug report
+about: Report incorrect or unexpected behavior of any cordis libraries/packages
+title: ''
+labels: 's: unverified, type: bug'
```

Comment:
We should probably re-do labels before we merge this and think about things a little.

Though these look good here.